src/rgw/setup/rgw_service.py

```diff
@@ -27,14 +27,14 @@ class RgwService:
     """Entrypoint class for RGW."""
 
     @staticmethod
-    def start(conf: MappedConf, config_file, log_file, motr_trace_dir, index: str = '1',):
+    def start(conf: MappedConf, config_file, log_file, motr_trace_dir, rgw_crash_dir, index: str = '1',):
```

Comment:
we should have defaults for all these arguments

Reply:
done. if this parameter is empty then we will use current working directory as default value